### DISCUSSION TITLE
Nette\Database\DriverException #45000

### DISCUSSION THREAD

**Discussion by m65v19** [OpenVK/openvk#300] [General] [answered]
Ошибка возникает каждый раз, когда нажимаю кнопку "Write" с сообщением на странице пользователя https://openvk.su/mikubot
![Screenshot_2021-11-21_04-34-13](https://user-images.githubusercontent.com/15930110/142744109-49e7c4e0-9d7f-488d-b048-77c47a23fe27.png)
 
Пишу сюда, т.к. было написано писать, но что такое "Чат ОВК" я не знаю. Кажется, в ОВК ещё нет чатов, только личка
![Screenshot_2021-11-21_04-33-23](https://user-images.githubusercontent.com/15930110/142744130-806f6879-ca66-461c-8666-28ca813a6a1e.png)
.

**Comment by ghost** [comment, marked as answer]
Проблема давным давно устранена, и она была связана с типом уведомлений, которого попросту не существовало.

"Чат OpenVK" - в нашем случае это чат в Telegram.
